pep-0570.rst

```diff
@@ -237,6 +343,16 @@ Whilst the followings are not::
     def name(p1=None, p2, /, p_or_kw=None, *, kw):
```

Comment:
I would probably use "While" instead of "Whilst" on folded line 234/340. As far as I know that's the correct way to use "whilst", but whilst is archaic and doesn't improve the clarity of the statement.